caql/readme.md

```diff
@@ -0,0 +1,3 @@
+# Circonus Grammar
+
+CaQLParser.g4 and Metrics.g4 are defined for parsing Circonus CaQL expression.
```

Comment:
You might want to add your name as author and license to your code (MIT or BSD or ...) in this file and/or grammar files.

Reply:
add BSD license and don't add author for now.

Reply:
hello @kaby76 , I fixed pom issues, test ok and added BSD license, please check.